CHANGELOG.md

```diff
@@ -10,6 +10,33 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ### Fixed
 
+## [0.39.1](https://github.com/returntocorp/semgrep/releases/tag/v0.39.1) - 2021-01-26
+
+No new changes in this version.
+This is a re-release of 0.39.0 due to an error in the release process.
+
+## [0.39.0](https://github.com/returntocorp/semgrep/releases/tag/v0.39.0) - 2021-01-26
+
+### Added
+
+- Support rules in JSON or JSonnet format (#2428).
```

Comment:
Well, actually those features are quite experimental and support only in semgrep-core.